src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -12,8 +12,8 @@
  */
 public class GuiSettings implements Serializable {
 
-    private static final double DEFAULT_HEIGHT = 600;
-    private static final double DEFAULT_WIDTH = 740;
+    private static final double DEFAULT_HEIGHT = 1000;
+    private static final double DEFAULT_WIDTH = 1500;
```

Comment:
minor but essential change to the window: we need a bigger space if we want to display more information